Libraries/Cordio/controller/sources/ble/lhci/lhci_cmd_vs.c

```diff
@@ -83,10 +92,76 @@ bool_t lhciCommonVsStdDecodeCmdPkt(LhciHdr_t *pHdr, uint8_t *pBuf)
     uint8_t evtParamLen = 1; /* default is status field only */
     uint32_t regReadAddr = 0;
 
+    static uint32_t * address;
+
     /* Decode and consume command packet. */
 
     switch (pHdr->opCode) {
         /* --- extended device commands --- */
+        
+
+    case LHCI_OPCODE_VS_DEVICE_RESET: {
+        char* pMsg = WsfMsgDataAlloc(1,1);
+        wsfHandlerId_t handlerId;
+        handlerId = WsfOsSetNextHandler(device_reset);
+        WsfMsgSend(handlerId, pMsg);
+
+        break;
+    }
+
+    case LHCI_OPCODE_VS_MEMORY_ERASE: {
+        LL_TRACE_INFO0("Erasing the memory...");
+        uint32_t addr;
+        uint32_t size;
+
+
+        BSTREAM_TO_UINT32(addr, pBuf);
+        address = (uint32_t*) addr;
+        uint8_t * addressErase = (uint8_t*) addr;
+
+        BSTREAM_TO_UINT32(size, pBuf);
+        volatile uint32_t address32 = (uint32_t)addressErase;
```

Comment:
What is addresss32 for? I only see it set

Reply:
> What is addresss32 for? I only see it set

Sorry, this becomes unused after several changes. Deleted.